dotcom-rendering/src/web/components/TableOfContents.tsx

```diff
@@ -79,8 +84,12 @@ export const TableOfContents = ({ tableOfContents }: Props) => {
 			</summary>
 
 			<ul>
-				{tableOfContents.map((item) => (
-					<li key={item.id} css={listItemStyles}>
+				{tableOfContents.map((item, index) => (
+					<li
+						key={item.id}
+						css={listItemStyles}
+						data-link-name={`toc-item-${index}-${item.id}`}
```

Comment:
Just a thought of using the `toc` namespace but `table-of-contents` parent name might be a little cognitive load for analysts

i.e. `table-of-contents-${index}-{item.id}` or calling the parent `toc` standardises it a little.

Reply:
Great idea, I changed them all to have `table-of-contents`